src/main/java/com/flightstats/hub/channel/ChannelLatestResource.java

```diff
@@ -5,14 +5,14 @@
 import com.flightstats.hub.dao.ChannelService;
 import com.flightstats.hub.model.*;
 import com.flightstats.hub.util.TimeUtil;
-import com.google.common.base.Optional;
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.UriInfo;
 import java.net.URI;
+import java.util.Optional;
```

Comment:
Yes. If functionality is available in java lib, we should definitely use that! 💯